### PR TITLE
fix: enable MD023

### PR DESCRIPTION
**Pull request by nschonni**
### Description

Rule is enabled upstream. There was one RU file that had some issues, so I sync'd with the en-US content and tryed to keep it to just the part of the page that hadn't really been translated yet

### CODE REVIEW

File: files/ru/web/api/webvtt_api/index.md [hunk 1]
```diff
@@ -104,62 +104,64 @@ NOTE Это последняя строка в файле
...
-```
+  NOTE This last line may not translate well.
 
-## Стилизация реплик WebTT
+  3
+  00:02:25.000 --> 00:02:30.000
+  - Ta en kopp
+  ```
 
-Реплики WebVTT можно стилизовать, используя псевдоэлемент {{cssxref("::cue")}}.
```
Comment (nschonni) [inline files/ru/web/api/webvtt_api/index.md:160]: Not sure if this matches the synced text, so I didn't revert it

File: files/ru/web/api/webvtt_api/index.md [hunk 2]
```diff
@@ -172,21 +174,21 @@ video::cue(b) {
 }
 ```
 
-В данном примере фоном видео будет серый градиент, с цветом текста `papayawhip`. Также, текст, выделенный жирным шрифтом с помощью элемента {{HTMLElement("b")}}, имеет цвет `peachpuff`.
```
Comment (nschonni) [inline files/ru/web/api/webvtt_api/index.md:175]: Same with this section

**Review by nschonni — COMMENTED** [review]
@sashasushko I wasn't sure if the 2 blocks highlighted below matched the sync'd text. If they do, I can try and add them back. I just copied the en-US content from where it looked like the translation in the page had stopped

File: files/ru/web/api/webvtt_api/index.md [hunk 3]
```diff
@@ -104,62 +104,64 @@ NOTE Это последняя строка в файле
...
+  3
+  00:02:25.000 --> 00:02:30.000
+  - Ta en kopp
+  ```
 
-Реплики WebVTT можно стилизовать, используя псевдоэлемент {{cssxref("::cue")}}.
+## Styling WebVTT cues
 
-### В CSS-стилях сайта
+You can style WebVTT cues by looking for elements which match the {{cssxref("::cue")}} pseudo-element.
```
Comment (sashasushko) [inline files/ru/web/api/webvtt_api/index.md:162]:
```suggestion
Реплики WebVTT можно стилизовать, используя псевдоэлемент {{cssxref("::cue")}}.
```

File: files/ru/web/api/webvtt_api/index.md [hunk 4]
```diff
@@ -172,21 +174,21 @@ video::cue(b) {
 }
 ```
 
-В данном примере фоном видео будет серый градиент, с цветом текста `papayawhip`. Также, текст, выделенный жирным шрифтом с помощью элемента {{HTMLElement("b")}}, имеет цвет `peachpuff`.
+Here, all video elements are styled to use a gray linear gradient as their backgrounds, with a foreground color of `"papayawhip"`. In addition, text boldfaced using the {{HTMLElement("b")}} element are colored `"peachpuff"`.
```
Comment (sashasushko) [inline files/ru/web/api/webvtt_api/index.md:177]:
```suggestion
В данном примере фоном видео будет серый градиент, с цветом текста `"papayawhip"`. Также, текст, выделенный жирным шрифтом с помощью элемента {{HTMLElement("b")}}, имеет цвет `"peachpuff"`.
```

File: files/ru/web/api/webvtt_api/index.md [hunk 5]
```diff
@@ -172,21 +174,21 @@ video::cue(b) {
...
 
-Фрагмент HTML, приведённый ниже, отображает видео.
+The HTML snippet below actually handles displaying the media itself.
```
Comment (sashasushko) [inline files/ru/web/api/webvtt_api/index.md:179]:
```suggestion
Фрагмент HTML, приведённый ниже, отображает видео.
```

**Pull request merged**